WeChat backup error, is not only a single backup, backup in quantity is also wrong. Backup APK is down, but won't the data backup, chat records, there is no beautiful MIUI system. Stop the lock screen function also is not normal

yphxp opened the pull request:
WeChat backup error, is not only a single backup, backup in quantity is also wrong. Backup APK is down, but won't the data backup, chat records, there is no beautiful MIUI system. Stop the lock screen function also is not normal

Justadudeinspace reviewed (commented):
Merge Forgitten Branch

Dziubek committed:
Translated using Weblate (Polish)

Currently translated at 46.2% (25 of 54 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/pl/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (55 of 55 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/uk/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (55 of 55 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/it/

JnBenites committed:
Translated using Weblate (Spanish)

Currently translated at 40.0% (22 of 55 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/es/

Commit:
Translated using Weblate (Latvian)

Currently translated at 5.4% (3 of 55 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/lv/

arthurzam committed:
Translated using Weblate (Hebrew)

Currently translated at 73.6% (333 of 452 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/he/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (461 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/uk/

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (461 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/et/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (461 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/it/

Commit:
Translated using Weblate (Latvian)

Currently translated at 100.0% (461 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/lv/

machiav3lli committed:
TargetSDK 36

machiav3lli committed:
Update Dependencies

machiav3lli committed:
8.3.14

Commit:
Translated using Weblate (Spanish)

Currently translated at 99.7% (460 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/es/

Dziubek committed:
Translated using Weblate (Polish)

Currently translated at 52.7% (29 of 55 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/pl/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (56 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/it/

Commit:
Translated using Weblate (Latvian)

Currently translated at 7.1% (4 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/lv/

Kefir2105 committed:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (56 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/uk/

samuel0220 committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 92.4% (426 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/pt_BR/

Commit:
Translated using Weblate (Japanese)

Currently translated at 1.7% (1 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/ja/

arthurzam committed:
Translated using Weblate (Hebrew)

Currently translated at 3.5% (2 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/he/

arthurzam committed:
Translated using Weblate (Hebrew)

Currently translated at 73.3% (338 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/he/

Trucbizar365 committed:
Translated using Weblate (French)

Currently translated at 96.3% (444 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/fr/

karigane-cha committed:
Translated using Weblate (Japanese)

Currently translated at 91.7% (423 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ja/

karigane-cha committed:
Translated using Weblate (Japanese)

Currently translated at 3.5% (2 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/ja/

sergiogomes13-ship-it committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 80.3% (45 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/pt_BR/

machiav3lli committed:
Update Dependencies

machiav3lli committed:
Fix Dependencies (compose.ui.test.*)

machiav3lli committed:
Add Dependency documentfile (as now is removed from material)

machiav3lli committed:
Update AGP

machiav3lli committed:
Update Gradle

machiav3lli committed:
Fix: Test using removed material icons

machiav3lli committed:
Update: Simplify ExpandingFadingVisibility

machiav3lli committed:
Update: Collect flow instead of forwarding viemodel in tools

machiav3lli committed:
Update: Move secondary functions out of BackendController to BackendUtils or PackageInfo extension

machiav3lli committed:
Update: Move function to add to global block to AppVM

machiav3lli committed:
Update: Split MainVM workload into 4 VMs

machiav3lli committed:
Update: Revamp CoroutinesAsyncTask and relevant classes

machiav3lli committed:
Update: Replace DB usage in repositories with the respective DAO

machiav3lli committed:
Update: Revamp PackageRepository abstracting BackupsCache and moving collection to VMs

machiav3lli committed:
Clean up

machiav3lli committed:
Add Nav3 Dependencies

machiav3lli committed:
Add: Nav3-based NavDisplay

machiav3lli committed:
Update: NavHost with the new NavDisplay

machiav3lli committed:
Add: SchedulerState to SchedulesVM and respective page

machiav3lli committed:
Add: ScheduleState to ScheduleVM and respective page

machiav3lli committed:
Clean up

machiav3lli committed:
Add: Random name for new schedules

korn3r committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (461 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ru/

Commit:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 97.8% (451 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/zh_Hans/

machiav3lli committed:
Add: CaretDownUp & CaretUpDown compose icons

machiav3lli committed:
Update: Revamp expandable block layout

machiav3lli committed:
Update: Revamp chip layouts

machiav3lli committed:
Update: Revamp action button layouts

machiav3lli committed:
Update: Revamp action chips layout

machiav3lli committed:
Update: Revamp sort filter sheet top bar

machiav3lli committed:
Update: Fix back-navigation icon on secondary pages

machiav3lli committed:
Fix: Searching on batch pages

machiav3lli committed:
Add: Selections' repository

machiav3lli committed:
Update: Rebase Home context menu to avoid injecting view models

machiav3lli committed:
Clean up

machiav3lli committed:
Update: Move repositories package one level up

machiav3lli committed:
Add: Functions to check available storage for backup

machiav3lli committed:
Add: Check if available storage enough for backup (closes #986)

machiav3lli committed:
Update: Revamp root missing page

machiav3lli committed:
Update: Info page links to point to codeberg where needed

machiav3lli committed:
Add /debug to .gitignore

machiav3lli committed:
Fix: Removing lock page from stack when unlocked

machiav3lli committed:
Update: Separate enabled and disabled schedule lists

machiav3lli committed:
Update Kotlin

machiav3lli committed:
Fix: Changes related to the new Kotlin version

machiav3lli committed:
Update: Make findBackups suspend (and its call chain) and cache file scans

machiav3lli committed:
Update: Remove animated background from all pages

machiav3lli committed:
Update: Make horizontal dividers thicker

machiav3lli committed:
Update: Improve shell script checks debugging

machiav3lli committed:
Fix: Reduce checkroot false-negatives

machiav3lli committed:
Add: Checkroot alt-variants

machiav3lli committed:
Update: Make Blocklist entity more consistent

machiav3lli committed:
Update: Improve blocklist repository flows

machiav3lli committed:
Update: Reduce fade-in animation duration

machiav3lli committed:
Update: Run ScheduleWork directly from ScheduleReceiver bridging ScheduleService out

machiav3lli committed:
Remove: ScheduleService

machiav3lli committed:
Clean up

machiav3lli committed:
Update: Make states compose-compiler-stable

machiav3lli committed:
Add: Compose-compiler metrics and reports

machiav3lli committed:
Update: Simplify package icon composable

Atalanttore committed:
Translated using Weblate (German)

Currently translated at 93.4% (431 of 461 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/de/

machiav3lli committed:
Update: Move notification channel creation into WorkHandler

machiav3lli committed:
Clean up

machiav3lli committed:
Update Dependencies

machiav3lli committed:
Add stability-analyzer lib

machiav3lli committed:
Add: RefreshBackups worker

machiav3lli committed:
Update: Replace singular calls with refresh worker

machiav3lli committed:
Update: Revamp BackupsCache and PackageRepository

machiav3lli committed:
Update: Reduce unnecessary suspending

machiav3lli committed:
Clean up

machiav3lli committed:
Update: Reduce recompositions of recyclers where possible

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (478 of 478 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/it/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (478 of 478 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/it/

Commit:
Translated using Weblate (Latvian)

Currently translated at 100.0% (478 of 478 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/lv/

machiav3lli committed:
Update: Move setting to exact schedules to advanced settings

machiav3lli committed:
Remove: Animated background related prefs

machiav3lli committed:
8.3.15

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 89.3% (427 of 478 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/pt_BR/

damjang committed:
Translated using Weblate (Slovenian)

Currently translated at 36.6% (175 of 478 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/sl/

kimjaechul committed:
Translated using Weblate (Korean)

Currently translated at 73.9% (354 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

Commit:
Translated using Weblate (Latvian)

Currently translated at 100.0% (479 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/lv/

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (479 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/et/

Commit:
Translated using Weblate (Estonian)

Currently translated at 3.5% (2 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/et/

Commit:
Translated using Weblate (Polish)

Currently translated at 57.1% (32 of 56 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/pl/

arthurzam committed:
Translated using Weblate (Hebrew)

Currently translated at 71.6% (343 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/he/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (479 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/it/

alpserhatt committed:
Translated using Weblate (Turkish)

Currently translated at 99.7% (478 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/tr/

alpserhatt committed:
Translated using Weblate (Turkish)

Currently translated at 38.5% (22 of 57 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/tr/

machiav3lli committed:
Add: Roadmap issue tempalte

Neebe3289 committed:
Translated using Weblate (Indonesian)

Currently translated at 21.0% (12 of 57 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/id/

Jenna-66 committed:
!fixup: Fix: Running ScheduleReceiver on reboot (closes #983)

Fixup for f7eaa9d91a0fde24f1a29a5a4c9b0d9557e74926.
Thread was never started.
To schedule the alarms, NeoBackup had to be opened at least once.

Signed-off-by: Jenna-66 <jenna@jennalody.de>

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (57 of 57 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/it/

Commit:
Translated using Weblate (Latvian)

Currently translated at 8.7% (5 of 57 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/lv/

machiav3lli committed:
Update: Move roadmap issue tempalte to be forgejo only

machiav3lli committed:
Update README: Replace Taiga project with Codeberg (closes #1007)

machiav3lli committed:
Add: Basic navigation call

machiav3lli committed:
Add: Overlay scenes for Detail-List and BottomSheet

machiav3lli committed:
Add: Nav3 scenes strategy

machiav3lli committed:
Add: BottomSheet entries to NavDisplay

machiav3lli committed:
Update: Replace BottomSheetScaffold with Nav3 bottom sheet navigation

machiav3lli committed:
Clean up

machiav3lli committed:
8.3.16

sergiogomes13-ship-it committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 93.1% (446 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/pt_BR/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (479 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (57 of 57 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/uk/

MicheDePainn committed:
Translated using Weblate (French)

Currently translated at 68.4% (39 of 57 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/fr/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (58 of 58 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/uk/

bommbomm34 committed:
Translated using Weblate (German)

Currently translated at 89.9% (431 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/de/

Commit:
Translated using Weblate (Latvian)

Currently translated at 10.3% (6 of 58 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/lv/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (58 of 58 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/it/

arthurzam committed:
Translated using Weblate (Hebrew)

Currently translated at 74.9% (359 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/he/

Commit:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 95.1% (456 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/zh_Hans/

BehemothXxx committed:
Translated using Weblate (Russian)

Currently translated at 99.5% (477 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ru/

machiav3lli committed:
Fix: Duplicate backups (possible racing condition)

machiav3lli committed:
Update: Improve reliability of running async functions in receivers

machiav3lli committed:
Update: Revamp search bar and the top bars

machiav3lli committed:
Update: Revamp search bar and the top bars

machiav3lli committed:
Clean up

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (479 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/et/

Commit:
Translated using Weblate (Spanish)

Currently translated at 37.9% (22 of 58 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/es/

juanlopezmartinez503-collab committed:
Added translation using Weblate (Hindi (Latin script))

juanlopezmartinez503-collab committed:
Added translation using Weblate (Hindi (Latin script))

Commit:
Translated using Weblate (German)

Currently translated at 90.8% (435 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/de/

machiav3lli committed:
Fix: Backup equality checks

machiav3lli committed:
Update Gradle

machiav3lli committed:
Update AGP (and respective build script changes)

machiav3lli committed:
Update: Screenshots

machiav3lli committed:
Update: Replace buttons boolean positive with ColoringState enum

Umeaboy committed:
Translated using Weblate (Swedish)

Currently translated at 57.4% (275 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/sv/

Umeaboy committed:
Translated using Weblate (Swedish)

Currently translated at 58.8% (282 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/sv/

machiav3lli committed:
Update: Add action button to permission items

machiav3lli committed:
Add: ExtrasTags DB view

machiav3lli committed:
Add: AppState flow to AppVM

machiav3lli committed:
Add: Existing tags suggestion in the adding dialog

machiav3lli committed:
Update: Revamp app page

Umeaboy committed:
Translated using Weblate (Swedish)

Currently translated at 8.6% (5 of 58 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/sv/

machiav3lli committed:
Update: Rebase RootFile on SuFile

machiav3lli committed:
Add: Fallback options for suGetOwnerGroupContext

machiav3lli committed:
Clean up

machiav3lli committed:
Update JVM to 21

machiav3lli committed:
Add: Setting to enable/disable storage check on backup action (closes #1014)

machiav3lli committed:
Update: Make Batch page's bottom bar floating

machiav3lli committed:
Update Dependencies

egnima committed:
Translated using Weblate (Croatian)

Currently translated at 85.3% (409 of 479 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/hr/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (482 of 482 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/it/

dpentx committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (482 of 482 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/tr/

Commit:
Translated using Weblate (Latvian)

Currently translated at 100.0% (482 of 482 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/lv/

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (482 of 482 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/et/

machiav3lli committed:
Update: Revamp refresh icon

machiav3lli committed:
Clean up

machiav3lli committed:
Fix: FAQ link in issue templates

machiav3lli committed:
Add: XML parsing tools

machiav3lli committed:
Add: WifiNetwork class and its parsing

machiav3lli committed:
Fix: Restore on wifi backups on newer Android versions

machiav3lli committed:
Add: KeepAndroidOpen banner

machiav3lli committed:
Update: Revamp backup item layout

machiav3lli committed:
Add: Backup sharing utils and cache

machiav3lli committed:
Add: Backup sharing button

machiav3lli committed:
Fix: Block list indicator on button

machiav3lli committed:
Fix: Restore Andy in placeholder icons

machiav3lli committed:
Add: Shortcuts management utils

machiav3lli committed:
Add: Launcher shortcuts handling and updates

machiav3lli committed:
Enable Gradle toolchain builds

machiav3lli committed:
Update AGP

machiav3lli committed:
Update Dependencies

machiav3lli committed:
8.3.17

damjang committed:
Translated using Weblate (Slovenian)

Currently translated at 92.7% (447 of 482 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/sl/

damjang committed:
Translated using Weblate (Slovenian)

Currently translated at 92.7% (447 of 482 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/sl/

damjang committed:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/sl/

Commit:
Translated using Weblate (Estonian)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/et/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/it/

Commit:
Translated using Weblate (Korean)

Currently translated at 72.8% (354 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

Umeaboy committed:
Translated using Weblate (Swedish)

Currently translated at 12.0% (7 of 58 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/sv/

Commit:
Translated using Weblate (Latvian)

Currently translated at 11.8% (7 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/lv/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (8 of 8 strings)

Translation: Neo Backup/Help
Translate-URL: https://hosted.weblate.org/projects/neo-backup/help/ta/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ta/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (59 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/ta/

mapi68 committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (59 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/it/

dpentx committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/tr/

Kefir2105 committed:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/uk/

Kefir2105 committed:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (59 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/uk/

Commit:
Translated using Weblate (Korean)

Currently translated at 89.3% (434 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

Coding-Young committed:
Translated using Weblate (Korean)

Currently translated at 89.3% (434 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

Coding-Young committed:
Translated using Weblate (Korean)

Currently translated at 89.3% (434 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

dpentx committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (8 of 8 strings)

Translation: Neo Backup/Help
Translate-URL: https://hosted.weblate.org/projects/neo-backup/help/tr/

Coding-Young committed:
Translated using Weblate (Korean)

Currently translated at 83.1% (404 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

cyphra committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/es/

cyphra committed:
Translated using Weblate (Spanish)

Currently translated at 37.2% (22 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/es/

Coding-Young committed:
Translated using Weblate (Czech)

Currently translated at 93.0% (452 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/cs/

Coding-Young committed:
Translated using Weblate (Korean)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/ko/

Umeaboy committed:
Translated using Weblate (Swedish)

Currently translated at 13.5% (8 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/sv/

cyphra committed:
Translated using Weblate (Spanish)

Currently translated at 37.2% (22 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/es/

jomurodovs committed:
Added translation using Weblate (Uzbek)

jomurodovs committed:
Added translation using Weblate (Uzbek)

jomurodovs committed:
Translated using Weblate (Uzbek)

Currently translated at 6.7% (4 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/uz/

machiav3lli committed:
Translated using Weblate (Spanish)

Currently translated at 37.2% (22 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/es/

grill-glitch committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 94.0% (457 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/zh_Hans/

jomurodovs committed:
Translated using Weblate (Uzbek)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/uz/

jomurodovs committed:
Translated using Weblate (Uzbek)

Currently translated at 100.0% (8 of 8 strings)

Translation: Neo Backup/Help
Translate-URL: https://hosted.weblate.org/projects/neo-backup/help/uz/

Umeaboy committed:
Translated using Weblate (Swedish)

Currently translated at 15.2% (9 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/sv/

youtsuhodev committed:
Translated using Weblate (French)

Currently translated at 92.5% (450 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/fr/

UniqueActive committed:
FAQ: Newest supported Android ver is latest stable

MicheDePainn committed:
Translated using Weblate (French)

Currently translated at 76.2% (45 of 59 strings)

Translation: Neo Backup/Store Information
Translate-URL: https://hosted.weblate.org/projects/neo-backup/store-information/fr/

arifpedia committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (486 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/id/

Coool committed:
Translated using Weblate (Latvian)

Currently translated at 99.7% (485 of 486 strings)

Translation: Neo Backup/Strings
Translate-URL: https://hosted.weblate.org/projects/neo-backup/strings/lv/

machiav3lli committed:
Add: Oboarding prefs page composable

machiav3lli committed:
Update: Revamp welcome page to handle onboarding

machiav3lli committed:
Update: Revamp permissions page to handle onboarding

machiav3lli committed:
Add: Main onboarding page

machiav3lli committed:
Clean up

machiav3lli committed:
Add: Package names to package selector dialogs

machiav3lli committed:
Remove: Encryption warning dialog (onboarding is enough)

machiav3lli committed:
Update Gradle

machiav3lli committed:
Update AGP

machiav3lli committed:
Update Dependencies

machiav3lli committed:
Fix: API changes on dependency update

machiav3lli committed:
Update: Revamp PrefDelegate to replace Pref

machiav3lli committed:
Add: User NeoPrefs

machiav3lli committed:
Add: NeoPrefs composables

machiav3lli committed:
Update: Replace legacy user prefs with NeoPrefs

machiav3lli committed:
Clean up

machiav3lli committed:
Fix: Updating app on changing theme contrast preference

machiav3lli committed:
Fix: Non-reactive dependant NeoPrefs

machiav3lli committed:
8.3.18
